etc/apparmor/firejail-default

```diff
@@ -82,6 +82,9 @@ owner /proc/@{PID}/{uid_map,gid_map,setgroups} w,
 owner /proc/@{PID}/oom_score_adj w,
 owner /proc/@{PID}/clear_refs w,
 
+# Needed for torbrowser-launcher
+owner @{HOME}/.local/share/torbrowser/tbb/{i686,x86_64}/tor-browser_*/Browser/** ix,
```

Comment:
Before such rules were recommended only as user specific overrides in `firejail-local `policy. The idea behind current `firejail-default` policy is to block executing code from user writable paths, see [comment](https://github.com/netblue30/firejail/blob/970d679d85a60792373f3b324497e91414d1b716/etc/apparmor/firejail-default#L89):

> Allow running programs only from well-known system directories

Adding such override in `firejail-default` will self-circumvent that policy as it will be bypassed by any code put on writable path. For this reason I think firejail-default shouldn't be patched such way but torrowser-launcher and any other app that execute something from home should remove appamror instead (leaving it as user opt-in if they want to tweak firejail-local policy on their own).

Reply:
Thanks for the review. I agree it makes more sense to implement this as opt-in and adapted the PR accordingly.